frontend/src/pages/signup/Signup.tsx

```diff
@@ -30,8 +32,19 @@ export default function SignUpPage() {
     try {
       setError('');
       setLoading(true);
-      await signup(email, password);
+      const user = await signup(email, password);
+
```

Comment:
probably the branch is not up to date, merging dev branch into this one, should do the trick